lib/less/tree/import.js

```diff
@@ -127,7 +127,7 @@ Import.prototype.eval = function (context) {
         }
     }
 
-    if (this.options.inline) {
+    if (this.options.inline && !this.path.currentFileInfo.reference) {
```

Comment:
Original use case was without mixin:
````
// ref.less
@import (inline) "boo.css"
````

````
// master.less
@import (reference) "ref";
````

The boo.css would appear in output and should not. I will add test with mixin to make sure the fix wont break that - the boo.css should appear in output in that case since it was brought in from that mixin.